polaris-react/stylelint/plugins/coverage/index.js

```diff
@@ -0,0 +1,10 @@
+const stylelint = require('stylelint');
+
+const namespace = require('./utils/namespace');
+const rules = require('./rules');
+
+const rulesPlugins = Object.keys(rules).map((ruleName) => {
```

Comment:
```suggestion
const coveragePlugins = Object.keys(rules).map((ruleName) => {
```

Reply:
Though the naming isn't the best, seems like this is a common convention among other plugins

* [stylelint-order](https://github.com/hudochenkov/stylelint-order/blob/master/index.js#L5)
* [stylelint-a11y](https://github.com/YozhikM/stylelint-a11y/blob/master/src/index.js#L4)
* [stylelint-scss](https://github.com/stylelint-scss/stylelint-scss/blob/master/src/index.js#L5)
* [stylelint-plugin-stylus](https://github.com/ota-meshi/stylelint-plugin-stylus/blob/master/lib/index.js#L6)

Cool with sticking with this convention?

Reply:
`stylelint-polaris-coverage`